test/integration/reload_test.lua

```diff
@@ -99,6 +102,9 @@ function g.test_router()
 end
 
 function g.test_storage()
+    t.skip_if(pcall(require, 'cartridge.hotreload') == false,
```

Comment:
As I can see, there is a check (`pcall(require, 'cartridge.hotreload') == false`)  of `hotreload` support in several places in the code. Maybe it should be a separate function? BTW it is unlikely the realization of getting `hotreload` will change, but I would put 
```lua
function get_cartridge_hotreload()
    return pcall(require, 'cartridge.hotreload')
end
```
in a separate function.

Reply:
That's fair, reworked